packages/bot-skeleton/src/services/api/api-base.js

```diff
@@ -48,7 +48,7 @@ class APIBase {
     reconnectIfNotConnected = () => {
         // eslint-disable-next-line no-console
         console.log('connection state: ', this.api.connection.readyState);
-        if (this.api.connection.readyState !== 1) {
+        if (this.api.connection.readyState !== 1 || this.api.connection.readyState !== 0) {
```

Comment:
these are somehow the opposite of each other!
what do you want to achieve with this condition?

Reply:
When the connection state is going zero still it is calling for subscribe function and it is causing that error.So I added this condition.

Reply:
how many states do we have on it?

Reply:
We have 0,1,2,3 states

Reply:
@yaswanth-deriv this would always return true!
having this condition or removing it doesn't make any difference.
* For readyState equal to 0, the first part (readyState !== 1) is true, making the whole condition true.
* For readyState equal to 1, the second part (readyState !== 0) is true, making the whole condition true.
* For readyState equal to 2 or 3, both parts are true, making the whole condition true.

Reply:
unless your intention was to add `&&` instead of `||`

Reply:
Yeah sorry It has to be &&